include/seqan3/alphabet/aminoacid/all.hpp

```diff
@@ -55,38 +57,41 @@
+ * \par Conversions
+ * | Amino acid name            | Three letter code | One letter code | Remapped in\nseqan3::aa20  |
+ * |----------------------------|-------------------|-----------------|----------------------------|
+ * |    Alanine                 | Ala               | A               | A                          |
+ * |    Arginine                | Arg               | R               | R                          |
+ * |    Asparagine              | Asn               | N               | N                          |
+ * |    Aspartic acid           | Asp               | D               | D                          |
+ * |    Cysteine                | Cys               | C               | C                          |
+ * |    Glutamic acid           | Glu               | E               | E                          |
+ * |    Glutamine               | Gln               | Q               | Q                          |
+ * |    Glycine                 | Gly               | G               | G                          |
+ * |    Histidine               | His               | H               | H                          |
+ * |    Isoleucine              | Ile               | I               | I                          |
+ * |    Leucine                 | leu               | L               | L                          |
+ * |    Lysine                  | Lys               | K               | K                          |
+ * |    Methionine              | Met               | M               | M                          |
+ * |    Phenylalanine           | Phe               | F               | F                          |
+ * |    Proline                 | Pro               | P               | P                          |
+ * |    Serine                  | Ser               | S               | S                          |
+ * |    Threonine               | Thr               | T               | T                          |
+ * |    Tryptophan              | Trp               | W               | W                          |
+ * |    Tyrosine                | Tyr               | Y               | Y                          |
+ * |    Valine                  | Val               | V               | V                          |
+ * |    Selenocysteine          | Sec               | U               | C                          |
+ * |    Pyrrolysine             | Pyl               | O               | K                          |
+ * | Asparagine or aspartic acid| Asx               | B               | D                          |
+ * | Glutamine or glutamic acid | Glx               | Z               | E                          |
+ * |    Leucine or Isoleucine   | Xle               | J               | L                          |
+ * |    Unknown                 | Xaa               | X               | S                          |
+ * |    Stop Codon              | N/A               | TERMINATOR      | W                          |
```

Comment:
I would mark those entries in the last column that are not changed just as `-`, because that makes spotting the changes easier. or maybe use colours? What do you think?

Reply:
Also the "official" one-letter-code for the Stop Codon is `*`. The paragraph below states that we use `::TERMINATOR` but I would keep the official things in this table.

Reply:
On second thought, maybe I will try the colors. I forgot that I grouped all of the extra alphabet letters at the bottom, so having two color groupings wouldn't be too bad.